microsoft/testsuites/dpdk/dpdkvpp.py

```diff
@@ -73,6 +73,7 @@ def run_test(self) -> None:
         vpp_detected_interface = (
             "GigabitEthernet" in vpp_interface_output
             or "VirtualFunctionEthernet" in vpp_interface_output
+            or "VSC" in vpp_interface_output
```

Comment:
Can you list the full outputs in the comment? I am not sure whether VSC_net is sriov type or not.

Reply:
The interface type when I tested last on SUSE was NetVSC0. I am pretty sure this corresponds to the net_vdev_netvsc PMD in dpdk, a newer PMD which makes use of the default hv_netvsc failover configuration. This would be the correct way to bind to that interface, older tests would have used net_failsafe since it was the only option but on DPDK >19.11 net_vdev_netvsc would be the way to go.

Reply:
(running some more tests to verify this)